### DISCUSSION TITLE
Filter for a posts in Public Squads

### DISCUSSION THREAD

**Discussion by vijaydeepak-vd** [dailydotdev/daily#1307] [Feature Requests]
Hi Team,
  I have written or shared some blogs in Public squad's where multiple people share their blogs. When there is a huge number of blogs in that squad, there is no filter to find the logged in user shared post.
Can you guys add any filter to find the logged in user shared post.

**Comment by nimrodkra** [comment]
@vijaydeepak-tt you can view everything you ever posted on the platform on your profile. Go to your profile and click the "Posts" tab. Here's mine for example: https://app.daily.dev/kramer/posts
